### PR TITLE
Add guidance to sidebar settings and secondary links pages

### PR DESCRIPTION
**Pull request by alex-ju**
This PR
 - adds guidance to sidebar settings page and fixes broken markup
 - adds guidance to secondary links page and fixes broken markup

[Trello card](https://trello.com/c/ij0U9BPn)

### CODE REVIEW

File: app/views/secondary_content_links/index.html.erb [hunk 1]
```diff
@@ -90,7 +81,7 @@
         <tr class="govuk-table__row">
           <td class="govuk-table__cell" colspan="3">
             <p class="govuk-hint">No secondary links have been added yet.</p>
-          </tr>
+          </td>
```
Comment (kevindew) [inline app/views/secondary_content_links/index.html.erb]: 🕵 👍

File: app/views/navigation_rules/edit.html.erb [hunk 2]
```diff
@@ -20,6 +20,12 @@
 
 <%= render 'shared/steps/form_notice', message: notice, status: "success" %>
 
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds govuk-!-margin-bottom-6">
+    <%= render_markdown t("sidebar_settings.guidance_markdown") %>
+  </div>
+</div>
```
Comment (kevindew) [inline app/views/navigation_rules/edit.html.erb]: Not sure if I should comment to you on this as the implementer but this copy feels a bit out of context when you browse this page.

Screen Shot 2019-10-14 at 15 52 09

It feels like this is missing something that introduces the options - or maybe the explanations should be after the table? I get though that we're in a bit of a pickle as there isn't a spot for contextual guidance.

If it's all product ok'd happy to tick it though as code looks good.

File: config/locales/en.yml [hunk 3]
```diff
@@ -90,14 +90,13 @@ en:
 
   secondary_content_links:
     guidance_markdown: |
-      ##What is secondary content?
```
Comment (kevindew) [inline config/locales/en.yml]: This looks loads better without this question

File: config/locales/en.yml [hunk 4]
```diff
@@ -90,14 +90,13 @@ en:
...
-
       Secondary content is content which is not linked to in the step by step, but has the step by step sidebar showing on it.
 
       We use it for content which:
 
-      - could help you complete a task but is not the main piece of content included in the step by step
+      - could help you complete a task but is not the main piece of content included in the step by step.
```
Comment (kevindew) [inline config/locales/en.yml]: Do you mean for the list item below to have a full stop too? Our styleguide tends to not have the periods at the end of list items: https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#bullet-points-steps

Comment (alex-ju) [reply config/locales/en.yml]: Good spot. I've just pasted from the document without looking a the diff. Removed the full stop.

File: app/views/navigation_rules/edit.html.erb (hunk 2 above)
Comment (alex-ju) [reply app/views/navigation_rules/edit.html.erb]: Agree with you that it feels a bit out of the blue, can raise it with product to see if they can enhance it.

**Pull request merged**